src/app/screens/settings/privacyPreferences/index.tsx

```diff
@@ -0,0 +1,101 @@
+import styled, { useTheme } from 'styled-components';
+import BottomBar from '@components/tabBar';
+import TopRow from '@components/topRow';
+import { useNavigate } from 'react-router-dom';
+import { useTranslation } from 'react-i18next';
+import Switch from 'react-switch';
+import { useEffect, useState } from 'react';
+import { hasOptedInMixPanelTracking, optInMixPanel, optOutMixPanel } from '@utils/mixpanel';
+import useWalletSelector from '@hooks/useWalletSelector';
+
+const Container = styled.div((props) => ({
+  display: 'flex',
+  flexDirection: 'column',
+  flex: 1,
+  overflowY: 'auto',
+  padding: props.theme.spacing(8),
+  fontSize: '0.875rem',
+  color: props.theme.colors.white['200'],
+  '&::-webkit-scrollbar': {
+    display: 'none',
+  },
+}));
+
+const TextContiner = styled.div({
+  lineHeight: '140%',
+});
+
+const SwitchContainer = styled.div((props) => ({
+  display: 'flex',
+  justifyContent: 'space-between',
+  alignItems: 'center',
+  marginTop: props.theme.spacing(16),
+  fontWeight: 500,
+}));
+
+const CustomSwitch = styled(Switch)`
+  .react-switch-handle {
+    background-color: ${({ checked }) =>
+      checked ? '#FFFFFF' : 'rgba(255, 255, 255, 0.2)'} !important;
+    border: ${({ checked }) => (checked ? '' : '4px solid rgba(255, 255, 255, 0.2)')} !important;
+  }
+`;
```

Comment:
I see this component reused a lot. 

can we define it as a new item in @components/ui-library/switch.tsx, so that it's easier to update the switches app-wide later for the revamp?

Reply:
actually, don't worry about including in this PR. I'll include it in https://github.com/secretkeylabs/xverse-web-extension/pull/588